__tests__/AccountSettings.js

```diff
@@ -1,13 +1,13 @@
 import { getInstallationInfo } from 'model/services/giantSwarm';
+import { getConfiguration } from 'model/services/metadata';
```

Comment:
I don't know how the changes in these test files got in here, I cannot remember doing them. For now I suggest to ignore them. Or should I revert these?

Reply:
They're part of the version check. They are in master, probably appeared here after the merge

Reply:
ok :+1: